src/components/Footer.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const Footer: React.FC = () => {
+    return (
+        <>
+            <div className='mt-10 px-20 fixed bottom-0 w-full border-t py-3 bg-white'>
```

Comment:
If you have decided to add component as part of this PR, lets also cover the tests.
Ensure you have tests for any piece of code.

Reply:
I have added